rotkehlchen/db/search_assets.py

```diff
@@ -0,0 +1,119 @@
+from typing import TYPE_CHECKING, Any, Dict, List, Optional
+
+from polyleven import levenshtein
+
+from rotkehlchen.assets.types import AssetType
+from rotkehlchen.constants.assets import A_ETH, A_ETH2
+from rotkehlchen.constants.resolver import ChainID
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+    from rotkehlchen.db.filtering import AssetsFilterQuery
+
+
+def search_assets_levenshtein(
+        filter_query: 'AssetsFilterQuery',
+        db: 'DBHandler',
+        substring_search: str,
+        limit: Optional[int],
+        search_nfts: bool,
+) -> List[Dict[str, Any]]:
+    """Returns a list of asset details that match the search keyword using the Levenshtein distance approach."""  # noqa: E501
```

Comment:
You moved this function if I'm not wrong from the globaldb code but I don't see the diff removing it from there. Are you duplicating code?

Reply:
Ah yeah, thanks! I just forgot to delete it from there